Firebase/InstanceID/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-# Unreleased
-- [changed] Ensure tokenWithAuthorizedEntity:scope:options:handler method is refreshing token if token is not feshed any more. (#4096)
+# 2019-11-05 -- 4.2.7
+- [fixed] Fixed a crash in checkTokenRefreshPolicyWithIID: and ensure tokenWithAuthorizedEntity:scope:options:handler method is refreshing token if token is not feshed any more. (#4167)
```

Comment:
`feshed` -> `refreshed`?

Also, pleas encapsulate code (i.e. method names) in backticks ``

Reply:
Done.